cloud/keystone.go

```diff
@@ -228,6 +232,15 @@ func isTokenScopedToAdminProject(result *v3tokens.GetResult) bool {
 	return (err == nil) && s.IsAdminProject
 }
 
+func (client *keystoneV3Client) validateRolesForAdmin(roleIDs []string) error {
+	if len(roleIDs) != 1 || roleIDs[0] != schema.AdminRole {
+		client.updateCounter(1, "error.token.mismatched_role_admin")
+		return fmt.Errorf("Admin token should have only a single admin role instead of '%v'", roleIDs)
```

Comment:
nit: errors should start from small letter

Reply:
Probably they should but this file already uses capital letters for them, I wanted to be consistent.